tests/framework/types.go

```diff
@@ -53,6 +53,7 @@ type OsmTestData struct {
 
 	// Kind cluster related vars
 	ClusterName                    string // Kind cluster name (used if kindCluster)
+	ClusterOS                      string // The operating system of the working nodes in the cluster. Currently only linux is supported.
```

Comment:
I think it's okay to start out by scheduling these tests to work either on Linux workers or Windows workers, but we need to eventually include tests that validate Linux <-> Windows traffic

Reply:
I'd add a note here that testing mixed OS traffic is not yet supported.

Reply:
Also this var should be moved out of the "Kind cluster related vars" section since it's not applicable to kind

Reply:
I agree, it'll be better to have a few dedicated tests to handle mixed OS traffic and leave the rest as is.

Reply:
fixed with the latest update